cmd/sfncli/sfncli.go

```diff
@@ -149,11 +144,21 @@ func main() {
 			var taskCtxCancel context.CancelFunc
 			taskCtx, taskCtxCancel = context.WithCancel(mainCtx)
 
+			// register active state
```

Comment:
suggestion: could this just be a function in CloudwatchReporter that takes a context? It'll set the state to true and then set it to false when the context is Done?

Reply:
💯

Reply:
5b7b601